### PR TITLE
Update Server Software

### PR DESCRIPTION
**Pull request by DrAGoniPenguin**
I know you guys have like a dozen kb jars, but why not?

Description
What did you add? or change?
I have added KSpigot and ReliableSpigot.
The first is a custom knockback software, while the latter is a cannon jar.

I know of a jar called Catalyst, and it was made by someone on the StellarSpigot team I believe. I searched for it a year ago, but don't remember where it was. (somewhere on mcmarket) (The crack on bs is sort of suspicious, and it is rather heavy to decomp.)

Checklist
[✔ ] My contributions follows the style guidelines of this project
[✔ ] I have performed a self-review of my own contributions (checked for spelling errors, proper linking, etc)
[ ✔] I have run linting on my branch before submitting the PR

### CODE REVIEW

**Comment by DrAGoniPenguin** [comment]
The reason why the Lantern is highlighted is because I accidentally created a fork and merged with my own fork (I'm dumb), so I copied and pasted from my fork to this merge.

File: server/info/lists/server_software.md [hunk 1]
```diff
@@ -212,9 +212,20 @@ Currently Inactive
 Version: 1.16 - 1.17  
 Currently Active
 
+- [KSpigot](https://www.mc-market.org/resources/12302/) - KSpigot is a performance server jar built off of Paper that has custom knockback and entity pathing.
+Version: 1.12.2
```
Comment (JustDoom) [inline server/info/lists/server_software.md:216]:
```suggestion
Version: 1.12.2  
```

File: server/info/lists/server_software.md [hunk 2]
```diff
@@ -212,9 +212,20 @@ Currently Inactive
 Version: 1.16 - 1.17  
 Currently Active
 
+- [KSpigot](https://www.mc-market.org/resources/12302/) - KSpigot is a performance server jar built off of Paper that has custom knockback and entity pathing.
```
Comment (JustDoom) [inline server/info/lists/server_software.md:215]:
```suggestion
- [KSpigot](https://www.mc-market.org/resources/12302/) - KSpigot is a performance server jar built off of Paper that has custom knockback and entity pathing.  
```

File: server/info/lists/server_software.md [hunk 3]
```diff
@@ -212,9 +212,20 @@ Currently Inactive
...
+
 - [Lantern](https://github.com/LanternPowered/Lantern) - Lantern is a server that implements the SpongeAPI.  
 Version: 1.12 - 1.14, 1.16  
 Currently Inactive
+@@ -400,6 +404,9 @@ Discontinued
+Version: ???  
+Discontinued
+
+- [ReliableSpigot](https://www.mc-market.org/resources/15892/) - ReliableSpigot is a factions cannoning server jar with a heavily customizable faction related config. 
+Version: 1.8
```
Comment (JustDoom) [inline server/info/lists/server_software.md:227]:
```suggestion
Version: 1.8  
```

File: server/info/lists/server_software.md [hunk 4]
```diff
@@ -212,9 +212,20 @@ Currently Inactive
...
+Currently Active
+
 - [Lantern](https://github.com/LanternPowered/Lantern) - Lantern is a server that implements the SpongeAPI.  
 Version: 1.12 - 1.14, 1.16  
 Currently Inactive
+@@ -400,6 +404,9 @@ Discontinued
+Version: ???  
+Discontinued
+
+- [ReliableSpigot](https://www.mc-market.org/resources/15892/) - ReliableSpigot is a factions cannoning server jar with a heavily customizable faction related config.
```
Comment (JustDoom) [inline server/info/lists/server_software.md:226]:
```suggestion
- [ReliableSpigot](https://www.mc-market.org/resources/15892/) - ReliableSpigot is a factions cannoning server jar with a heavily customizable faction related config.  
```

**Review by JustDoom — APPROVED** [review]

File: server/info/lists/server_software.md [hunk 5]
```diff
@@ -212,9 +212,20 @@ Currently Inactive
...
 - [Lantern](https://github.com/LanternPowered/Lantern) - Lantern is a server that implements the SpongeAPI.  
 Version: 1.12 - 1.14, 1.16  
 Currently Inactive
+@@ -400,6 +404,9 @@ Discontinued
+Version: ???  
+Discontinued
+
+- [ReliableSpigot](https://www.mc-market.org/resources/15892/) - ReliableSpigot is a factions cannoning server jar with a heavily customizable faction related config.  
+Version: 1.8  
+Currently Active (?)
```
Comment (JustDoom) [inline server/info/lists/server_software.md:228]:
```suggestion
Currently Active
```

File: server/info/lists/server_software.md [hunk 6]
```diff
@@ -212,9 +212,20 @@ Currently Inactive
...
 Currently Active
 
+- [KSpigot](https://www.mc-market.org/resources/12302/) - KSpigot is a performance server jar built off of Paper that has custom knockback and entity pathing.  
+Version: 1.12.2  
+Currently Active
+
 - [Lantern](https://github.com/LanternPowered/Lantern) - Lantern is a server that implements the SpongeAPI.  
 Version: 1.12 - 1.14, 1.16  
 Currently Inactive
+@@ -400,6 +404,9 @@ Discontinued
```
Comment (JustDoom) [inline server/info/lists/server_software.md:222]:
```suggestion

```

**Comment by JustDoom** [comment]
had some weird stuff going on

**Pull request merged**